tool/bots.sh

```diff
@@ -95,7 +95,8 @@ if [ "$BOT" = "main" ]; then
     repo_tool repo-check
 
     # Analyze the source.
-    dart analyze --fatal-infos
+    dart analyze --fatal-infos lib
```

Comment:
duplicated line

Reply:
Also, why just analyze the source in `lib/`? Don't we want all the source in devtools_app?

If there are some sub-dirs we want to exclude we can do that w/ some analysis options surgery.

Reply:
good catch, this change accidentally leaked in. reverting.